synapse/config/server.py

```diff
@@ -228,7 +229,7 @@ def default_config(self, server_name, **kwargs):
           # Turn on the twisted ssh manhole service on localhost on the given
           # port.
           # - port: 9000
-          #   bind_address: 127.0.0.1
+          #   bind_address: ::1
```

Comment:
Apparently binding to ::1 does not bind to 127.0.0.1. I’ve tried above (I use nginx as a reverse proxy), and bind_addresses: ['::1'] did not bind 127.0.0.1, but ['::1', '127.0.0.1'] worked.